recipes/pastrami/meta.yaml

```diff
@@ -0,0 +1,140 @@
+{% set name = "pastrami" %}
+{% set version = "0.9.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/healthdisparities/pastrami/archive/v{{ version }}.tar.gz"
+  sha256: 4457ad6cda7a2384a1b22109caf3edbff191b3e0a8749f779525276424b4c55c
+
+build:
+  noarch: python
+  number: '0'
+  string: py_0
+
+requirements:
+  build:
+    - python >=3.8
+    - plink2
+    - setuptools
+    - _libgcc_mutex =0.1= conda_forge
+    - _openmp_mutex =4.5= 2_gnu
+    - brotlipy =0.7.0 =py38h0a891b7_1005
+    - bzip2 =1.0.8 =h7f98852_4
+    - ca-certificates =2022.12.7 =ha878542_0
+    - certifi =2022.12.7 =pyhd8ed1ab_0
+    - cffi =1.15.1 =py38h4a40e3a_3
+    - charset-normalizer =2.1.1=pyhd8ed1ab_0
+    - cryptography =39.0.2 =py38h3d167d9_0
+    - dill =0.3.6 =pyhd8ed1ab_1
+    - idna =3.4 =pyhd8ed1ab_0
+    - ld_impl_linux-64 =2.40 =h41732ed_0
+    - libblas =3.9.0 =16_linux64_openblas
+    - libcblas =3.9.0 =16_linux64_openblas
+    - libffi =3.4.2 =h7f98852_5
+    - libgcc-ng =12.2.0 =h65d4601_19
+    - libgfortran-ng =12.2.0 =h69a702a_19
+    - libgfortran5 =12.2.0 =h337968e_19
+    - libgomp =12.2.0 =h65d4601_19
+    - liblapack =3.9.0 =16_linux64_openblas
+    - libnsl =2.0.0 =h7f98852_0
+    - libopenblas =0.3.21 =pthreads_h78a6416_3
+    - libsqlite =3.40.0 =h753d276_0
+    - libstdcxx-ng =12.2.0 =h46fd767_19
+    - libuuid =2.32.1 =h7f98852_1000
+    - libzlib =1.2.13 =h166bdaf_4
+    - multiprocess =0.70.14 =py38h0a891b7_3
+    - ncurses =6.3 =h27087fc_1
+    - numpy =1.24.2 =py38h10c12cc_0
+    - openssl =3.0.8 =h0b41bf4_0
+    - packaging =23.0 =pyhd8ed1ab_0
+    - pandas =1.5.3 =py38hdc8b05c_0
+    - pathos =0.3.0 =pyhd8ed1ab_0
+    - pip =23.0.1 =pyhd8ed1ab_0
+    - platformdirs =3.1.0 =pyhd8ed1ab_0
+    - pooch =1.7.0 =pyhd8ed1ab_0
+    - pox =0.3.2 =pyhd8ed1ab_0
+    - ppft =1.7.6.6 =pyhd8ed1ab_0
+    - pycparser =2.21 =pyhd8ed1ab_0
+    - pyopenssl =23.0.0 =pyhd8ed1ab_0
+    - pysocks =1.7.1 =pyha2e5f31_6
+    - python-dateutil =2.8.2 =pyhd8ed1ab_0
+    - python_abi =3.8 =3_cp38
+    - pytz =2022.7.1 =pyhd8ed1ab_0
+    - readline =8.1.2 =h0f457ee_0
+    - requests =2.28.2 =pyhd8ed1ab_0
+    - scipy =1.10.1 =py38h10c12cc_0
+    - setuptools =67.5.1 =pyhd8ed1ab_0
+    - six =1.16.0 =pyh6c4a22f_0
+    - tk =8.6.12 =h27826a3_0
+    - typing-extensions =4.4.0 =hd8ed1ab_0
+    - typing_extensions =4.4.0 =pyha770c72_0
+    - urllib3 =1.26.14 =pyhd8ed1ab_0
+    - wheel =0.38.4 =pyhd8ed1ab_0
+    - xz =5.2.6 =h166bdaf_0
+  run:
```

Comment:
You should not restrict the dependencies like that if not absolutely necessary, and you should also only list the direct dependencise. Otherwise, it is highly likely that nothing else can be installed along this package into a Conda environment. Also, sometimes individual builds of packages get deleted from anaconda.org because they have a security issue or so, and if this happens, then your package would no longer be installable.

Normally, you should only specify minimum versions (if at all).

Reply:
Done